### PR TITLE
Fix Istio installation in e2e tests

### PR DESCRIPTION
**Pull request by torchiaf**
### Summary

It merges Istio and Prometheus e2e tests since Istio requires a Prometheus operator to be installed, then verified.

### Technical notes summary

`contains` function is bound to the `.accordion` divs so it's responsive to DOM changes.

- Istio and Prometheus tests merged in one file.
- Re-enable Istio nav item check and fixed installation.
- Added POs for 'Installed apps' page.

### Checklist
- [x] The PR is linked to an issue, or one is not required. The linked issue has a Milestone
- [x] The PR has a Milestone 
- [x] The PR template below has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests. The linked issue has appropriate QA labels
- [x] The PR has reviewed with UX (if required) and tested in light and dark mode

### CODE REVIEW

File: cypress/e2e/po/lists/chart-installed-apps.po.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+
+/**
+ * List component for home page cluster resources
```
Comment (richard-cox) [inline cypress/e2e/po/lists/chart-installed-apps.po.ts:4]: comment needs updating

File: cypress/globals.d.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ declare global {
       createAwsCloudCredentials(nsName: string, cloudCredName: string, defaultRegion: string, accessKey: string, secretKey: string): Chainable;
 
       getRancherResource(prefix: 'v3' | 'v1', resourceType: string, resourceId?: string, expectedStatusCode?: number): Chainable;
-      setRancherResource(prefix: 'v3' | 'v1', resourceType: string, resourceId: string, body: string): Chainable;
+      setRancherResource(prefix: 'v3' | 'v1', resourceType: string, resourceId: string, body: any): Chainable;
```
Comment (richard-cox) [inline cypress/globals.d.ts:45]: this could remain as string?

File: cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts [hunk 3]
```diff
@@ -186,4 +189,42 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
       });
     });
   });
+
+  describe('Istio', () => {
+    const istioVersion = '103.0.0%2Bup1.18.2';
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts:194]: Comment only. At some point we'll need to change this to navigate to the chart detail page first and install from there to get the latest available version. Monitoring above uses this process (`chartsPage.goToInstall()`)

File: cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts [hunk 4]
```diff
@@ -186,4 +189,42 @@ describe('Charts', { tags: ['@charts', '@adminUser'] }, () => {
       });
     });
   });
+
+  describe('Istio', () => {
```
Comment (richard-cox) [inline cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts:193]: If istio is now dependent on Prometheus could we add a comment here about it?

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts (hunk 4 above)
Comment (torchiaf) [reply cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts:193]: There is a comment below:

```
  describe('Istio local provisioning', () => {
    /**
     * Istio requires Prometheus operator to be installed, see previous steps.
     */
 ```
 
 I can move it above.

Comment (richard-cox) [reply cypress/e2e/tests/pages/charts/monitoring-istio.spec.ts:196]: oops, sorry, didn't spot. this location is fine

**Review by richard-cox — APPROVED** [review]

**Pull request merged**